egui_web/Cargo.toml

```diff
@@ -44,7 +45,7 @@ default = ["default_fonts"]
 default_fonts = ["egui/default_fonts"]
 persistence = ["egui/persistence", "ron", "serde"]
 screen_reader = ["tts"] # experimental
-
+use_glow_painter =["egui_glow/painter_only","egui_glow/epi"]
```

Comment:
I think just `glow` or `glow_painter` is a better name.

Please don't remove the blank lines - I think they aid in the readability of a file. If you don't like them, configure your editor to don't show them or something :)